src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2378,116 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
+                    stop_sequences=inference_parameters.get("stop_sequences", ["\n"]),
+                    sample_train=inference_parameters.get("sample_train", True),
+                    multi_label=inference_parameters.get("multi_label", False),
+                )
+            else:
+                adapter_spec = AdapterSpec(
+                    method=prompt_setting.method,
+                    instructions=prompt_setting.instructions,
+                    input_prefix=prompt_setting.input_prefix,
+                    input_suffix=prompt_setting.input_suffix,
+                    output_prefix=prompt_setting.output_prefix,
+                    output_suffix=prompt_setting.output_suffix,
+                    # Separate is basically language modeling, so can't easily use in-context examples
+                    max_train_instances=inference_parameters.get("max_train_instances", 5),
+                    num_outputs=1,
+                    max_tokens=0,
+                    temperature=inference_parameters.get("temperature", 0.0),
+                    sample_train=inference_parameters.get("sample_train", True),
+                )
+            metric_specs = get_exact_match_metric_specs()
+            if task in ["fact_checking", "bias"]:
+                metric_specs += get_multiple_choice_classification_metric_specs()
+        elif prompt_setting.method == ADAPT_GENERATION:
+            adapter_spec = AdapterSpec(
+                method=prompt_setting.method,
+                instructions=prompt_setting.instructions,
+                input_prefix=prompt_setting.input_prefix,
+                input_suffix=prompt_setting.input_suffix,
+                output_prefix=prompt_setting.output_prefix,
+                output_suffix=prompt_setting.output_suffix,
+                max_train_instances=inference_parameters.get("max_train_instances", 5),
+                num_outputs=inference_parameters.get("num_outputs", 1),
+                max_tokens=inference_parameters.get("max_tokens", 20),
+                temperature=inference_parameters.get("temperature", 0.0),
+                stop_sequences=inference_parameters.get("stop_sequences", ["\n"]),
+                sample_train=inference_parameters.get("sample_train", True),
+                multi_label=inference_parameters.get("multi_label", True),
+            )
+            metric_specs = (
+                get_cleva_generative_task_metric_spec(task, subtask) + get_cleva_generative_harms_metric_specs()
+            )
+        else:
+            raise ValueError(
+                f"{task} can only be {ADAPT_GENERATION}, {ADAPT_MULTIPLE_CHOICE_JOINT}, "
+                f"{ADAPT_MULTIPLE_CHOICE_SEPARATE_CALIBRATED} or {ADAPT_MULTIPLE_CHOICE_SEPARATE_ORIGINAL}"
+            )
+
+    return RunSpec(
+        name=run_spec_name,
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=metric_specs,
+        groups=["cleva"],
```

Comment:
Note that you will need to edit `schema.yaml` to add the new scenarios and metrics. I would recommend doing this in a new Pull Request rather than in this Pull Request, because editing `schema.yaml` is rather complicated.

Reply:
Thanks for pointing it out. We will edit `schema.yaml` in a new Pull Request.

Reply:
I've started a template for the schema.yaml in this PR #1845. I will get it merged soon, and then it would be great to have your help in checking it and filling it in.